src/main/java/com/bina/lrsim/simulator/ParallelSimulator.java

```diff
@@ -45,15 +48,17 @@ public static long process(RandomFragmentGenerator rfg, String outDir, String mo
     final String movieName;
     final SamplesDrawer samples;
     final int targetNumBases;
+    final SimulatorDriver.ModuleOptions options;
     final Spec spec;
     final int seed;
 
-    private Worker(Simulator sim, String outDir, String movieName, SamplesDrawer samples, int targetNumBases, Spec spec, int seed) {
+    private Worker(Simulator sim, String outDir, String movieName, SamplesDrawer samples, int targetNumBases, SimulatorDriver.ModuleOptions options, Spec spec, int seed) {
```

Comment:
should not change this, since this is public. Instead, use a new Spec to specify a new output format with adapter in it, then dispatch wrt to Spec within the implementation in Simulator